index.html

```diff
@@ -1,13 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This a portfolio of a full stack web developer">
+    <meta name="keywords" content="porfotlio, software developer, web developer">
+    <meta name="author" content="Julio Quezada">
     <link rel="stylesheet" href="./assets/css/style.css">
     <title>Document</title>
 </head>
+
 <body>
-    <h1 class="makeHeaderRed">My portfolio</h1>
+    <header>
+        <nav class="header-container">
+            <div class="logo">
+                <a href="#"><img src="./assets/images/logo3.png" alt="logo"></a>
+            </div>
+            <div class="hamburgerMenu">
+                <ul class="nav-items">
+                    <li class="list-item"><a href="#">Porfolio</a></li>
+                    <li class="list-item"><a href="#">About</a></li>
+                    <li class="list-item"><a href="#">Contact</a></li>
+                    <li class="list-item"><a href="#"><img src="./assets/images/mail.png" alt="mail logo"></a></li>
+                </ul>
+            </div>
+            <button type="button" class="hamburger-icon">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </button>
+        </nav>
```

Comment:
- [ ] The convention in software development is that the `nav` tags should be containers for navigation links that can redirect users to other parts of a webpage or other sites. 👍In your code, you used it as a container for the logo and the navigation links which semantically should be the header tag. Your code should be in the structure below:

```HTML
<header>
          <div class="logo">
              <a href="#"><img src="./assets/images/logo3.png" alt="logo"></a>
          </div>
          <nav class="header-container">
            <div class="hamburgerMenu">
                <ul class="nav-items">
                    <li class="list-item"><a href="#">Porfolio</a></li>
                    <li class="list-item"><a href="#">About</a></li>
                    <li class="list-item"><a href="#">Contact</a></li>
                    <li class="list-item"><a href="#"><img src="./assets/images/mail.png" alt="mail logo"></a></li>
                </ul>
            </div>
            <button type="button" class="hamburger-icon">
                <span class="bar"></span>
                <span class="bar"></span>
                <span class="bar"></span>
            </button>
        </nav>
</header>
```

#### Hint: It would be nice to use the `flexbox` on the `header` tag since that is the container for this part of the page. 👍